LICENSE

```diff
@@ -0,0 +1,27 @@
+Z3-Noodler
+
+This project is an SMT string solver, a modified version of Z3 SMT Solver, distributed under the MIT License.
+Original Z3 repository: https://github.com/Z3Prover/z3
+
+MIT License
+
+Copyright (c) Microsoft Corporation
```

Comment:
Is this correct? It seems like noodler is copyrighted under microsoft

Reply:
Both approaches are valid. The sequence of copyrights represents the derivative work which Z3-Noodler is. The original Z3 is copyrighted under Microsoft. Therefore, the original copyright needs to remain there, too.

Reply:
Ok, but we should make it more obvious, for me it looks now that microsoft is endorsing noodler development. Maybe say that Z3 is copyrighted under microsoft, and noodler is copyrighted by us?

Reply:
ah ok, I didn't know that